runtimes/standalone/src/lib.rs

```diff
@@ -380,9 +379,9 @@ parameter_types! {
 
 impl did::Config for Runtime {
 	type DidIdentifier = DidIdentifier;
-	type Event = Event;
-	type Call = Call;
-	type Origin = Origin;
+	type RuntimeEvent = RuntimeEvent;
+	type Call = RuntimeCall;
+	type Origin = RuntimeOrigin;
```

Comment:
I am not sure at this point if keeping `Origin` makes sense anymore, or if we should also rename this to `RuntimeOrigin`.

Reply:
fde113cb1b81146770db9d3edfab89e40100617a